sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -219,99 +219,99 @@ def test_init_port_sfp_status_tbl(self):
+                                                                              'nic_lane2_precursor2': '7',
+                                                                              'nic_lane2_maincursor': '7',
+                                                                              'nic_lane2_postcursor1': '7',
+                                                                              'nic_lane2_postcursor2': '7',
+                                                                              'tor_self_lane1_precursor1': '17',
+                                                                              'tor_self_lane1_precursor2': '17',
+                                                                              'tor_self_lane1_maincursor': '17',
+                                                                              'tor_self_lane1_postcursor1': '17',
+                                                                              'tor_self_lane1_postcursor2': '17',
+                                                                              'tor_self_lane2_precursor1': '7',
+                                                                              'tor_self_lane2_precursor2': '7',
+                                                                              'tor_self_lane2_maincursor': '7',
+                                                                              'tor_self_lane2_postcursor1': '7',
+                                                                              'tor_self_lane2_postcursor2': '7',
+                                                                              'tor_peer_lane1_precursor1': '7',
+                                                                              'tor_peer_lane1_precursor2': '7',
+                                                                              'tor_peer_lane1_maincursor': '17',
+                                                                              'tor_peer_lane1_postcursor1': '7',
+                                                                              'tor_peer_lane1_postcursor2': '17',
+                                                                              'tor_peer_lane2_precursor1': '7',
+                                                                              'tor_peer_lane2_precursor2': '7',
+                                                                              'tor_peer_lane2_maincursor': '17',
+                                                                              'tor_peer_lane2_postcursor1': '7',
+                                                                              'tor_peer_lane2_postcursor2': '17'}))
     def test_post_port_mux_static_info_to_db(self):
         logical_port_name = "Ethernet0"
-        mux_tbl = Table("state_db", "mux_static_info_tbl")
+        mux_tbl = Table("STATE_DB", "MUX_CABLE_STATIC_INFO")
```

Comment:
Suggest defining and using the following in y_cable_helper.py:

```python
MUX_CABLE_STATIC_INFO_TABLE = "MUX_CABLE_STATIC_INFO"
```

and referencing it here the other suggestions above (`xcvrd.y_cable_helper.MUX_CABLE_STATIC_INFO_TABLE `)

Reply:
fixed